pkg/util/merge/merge_test.go

```diff
@@ -588,20 +585,6 @@ func TestMergeVolumeAddVolume(t *testing.T) {
 	assert.Equal(t, corev1.EmptyDirVolumeSource{}, *volume1.EmptyDir)
 }
 
-func TestMergeVolumeMounts(t *testing.T) {
```

Comment:
this is actually the incorrect merging behaviour - uniqueness should be determined by name, path and subpath